src/lib/bootloader/bootloader_base.rb

```diff
@@ -99,6 +101,24 @@ def merge(other)
 
   private
 
+    def propose_resume
```

Comment:
as it is not specific to bootloader and it is more about finding suitable resume swap partition I would move whole code to `Yast::BootStorage`

Reply:
Renamed